pkgs/by-name/ve/vesktop/package.nix

```diff
@@ -24,7 +24,7 @@
 }:
 stdenv.mkDerivation (finalAttrs: {
   pname = "vesktop";
-  version = "1.5.0";
+  version = "1.5.1";
 
   src = fetchFromGitHub {
```

Comment:
you also need to update the hash of the `src`

Reply:
Done, thank you!